tests/baselines/reference/library-reference-4.symbols

```diff
@@ -12,10 +12,14 @@ declare var foo: any;
 
 === /node_modules/foo/node_modules/alpha/index.d.ts ===
 declare var alpha: any;
->alpha : Symbol(alpha, Decl(index.d.ts, 0, 11))
+>alpha : Symbol(alpha, Decl(index.d.ts, 0, 11), Decl(index.d.ts, 0, 11))

```

Comment:
These didnt have package.json so didnt get deduped. But I added test case 4b and 5b where there is package.json